avocado/utils/nfs.py

```diff
@@ -0,0 +1,349 @@
+"""
+Basic nfs support for Linux host. It can support the remote
+nfs mount and the local nfs set up and mount.
+"""
+import re
+import os
+import logging
+import commands
+
+from . import path
+from . import process
+from . import service
+
+from utils_conn import SSHConnection
+import utils_misc
+
+
+def nfs_exported():
+    """
+    Get the list for nfs file system already exported
+
+    :return: a list of nfs that is already exported in system
+    :rtype: a lit of nfs file system exported
+    """
+    exportfs = process.system_output("exportfs -v")
```

Comment:
Doesn't this require root? If so you should use `sudo = True` (everywhere where root is required)